src/less/section-title/base/section-title.less

```diff
@@ -0,0 +1,223 @@
+
+    > a {
+        border: 0;
+        padding: 0;
+    }
+
+    > a > span:not(.section-title__cta-icon) {
+        display: none;
+    }
+}
+
+.section-title__cta > a,
+.section-title__title > h2 > a {
+    color: @color-black;
+    text-decoration: none;
+
+    &:hover {
+        color: @color-black;
+        text-decoration: underline;
+    }
+
+    &:visited {
+        color: @color-black;
+    }
+}
+
+.section-title__cta-icon {
+    display: inline-block;
+    margin-left: 8px;
+
+    span& {
+        .background-icon-image-mixin('arrow-right-bold', 14px, 14px);
+    }
+
+    svg& {
+        height: 14px;
+        width: 14px;
+    }
+}
+
+.section-title__info {
+    margin: 0 24px 0 8px;
+
+    .infotip {
+        line-height: normal;
+    }
+
+    button.icon-btn.infotip__host {
+        height: 24px;
+        vertical-align: middle;
+        width: 24px;
+    }
+}
+
+.section-title__overflow {
+    margin: 0 24px 0 auto;
+
+    button.icon-btn {
+        height: 24px;
+        width: 24px;
+    }
+}
+
+.section-title__cta,
+.section-title__info,
+.section-title__overflow {
+    align-items: center;
+    display: flex;
+}
+
+@media (max-width: 600px) {
+    .section-title__title > h2 {
+        .typography-medium-bold();
+    }
+
+    .section-title__subtitle {
+        .typography-regular();
+    }
+
+    .section-title__cta {
+        font-size: @font-size-medium;
+        margin: 0;
+
+        & > a {
+            border: 0;
+            padding: 0;
+        }
+
+        & > a > span:not(.section-title__cta-icon) {
+            display: none;
+        }
+    }
+
+    .section-title__cta,
+    .section-title__info,
+    .section-title__overflow {
+        height: 24px;
+    }
+
+    .section-title.section-title--giant {
+        > .section-title__title > h2 {
+            .typography-large-2-bold();
+        }
+
+        > .section-title__cta,
+        > .section-title__info,
+        > .section-title__overflow {
+            height: 32px;
+        }
+    }
+
+    .section-title.section-title--large {
+        > .section-title__title > h2 {
+            .typography-large-1-bold();
+        }
+
+        > .section-title__cta,
+        > .section-title__info,
+        > .section-title__overflow {
+            height: 28px;
+        }
+    }
+
+    .section-title.section-title--small {
+        > .section-title__title > h2 {
+            .typography-regular-bold();
+        }
+
+        > .section-title__cta,
+        > .section-title__info,
+        > .section-title__overflow {
+            height: 20px;
+        }
+    }
+}
+
+@media (min-width: 601px) {
+    .section-title__title > h2 {
+        .typography-large-1-bold();
+    }
+
+    .section-title__subtitle {
+        .typography-medium();
+    }
+
+    .section-title__cta {
+        font-size: @font-size-large-1;
+    }
+
+    .section-title__cta,
+    .section-title__info,
+    .section-title__overflow {
+        height: 28px;
+    }
+
+    .section-title.section-title--giant {
```

Comment:
Do these all need to be chained? Can we just have `.section-title--giant`? Generally we try to avoid chained selectors.

Reply:
@ianmcburnie Change `.section-title.section-title--giant` to `.section-title--giant`, correct? Or are you also referring to the nested selectors inside `--giant {`?

Reply:
@ianmcburnie I changed `.section-title.section-title--giant` to `.section-title--giant`. If you had other changes in mind, please let me know.